plugin/plugins.go

```diff
@@ -74,36 +76,47 @@ func (c *PluginsProcessor) Load() error {
 // Run execute each in-tree or out-of-tree plugin by traversing the plugin list.
 func (c *PluginsProcessor) Run(cluster *v1.Cluster, phase Phase) error {
 	for _, config := range c.Plugins {
-		if ext := filepath.Ext(config.Name); ext == ".so" {
-			err := c.runOutOfTree(cluster, config, phase)
+		var p Interface
+		var err error
+		ext := filepath.Ext(config.Name)
+		if ext == ".so" {
+			p, err = c.getOutOfTreePlugin(config)
+			if err != nil {
+				return fmt.Errorf("failed to run plugin, %v", err)
```

Comment:
please distinguish the log as you can, since I found there are several places which have the same log prefix `failed to run plugin` . 

Otherwise when code runs into error, no one knows where happens the error.

Reply:
I will fix those problem in plugins v2.